src/main/java/dev/ebullient/convert/tools/JsonSourceCopier.java

```diff
@@ -50,8 +56,97 @@ public JsonNode handleCopy(T type, JsonNode copyTo) {
         return copyTo;
     }
 
-    protected JsonNode mergeNodes(T type, String copyToKey, JsonNode copyFrom, JsonNode copyTo) {
-        return copyTo;
+    // 	utils.js: static getCopy (impl, copyFrom, copyTo, templateData,...) {
+    protected JsonNode mergeNodes(T type, String originKey, JsonNode copyFrom, JsonNode copyTo) {
+        // edit in place: if you don't, lower-level copies will keep being revisted.
+        ObjectNode target = (ObjectNode) copyTo;
+
+        JsonNode _copy = MetaFields._copy.getFrom(copyTo);
+        // normalizeMods needs to be called before we handle the trait, but handle trait may modify the mod, so
```

Comment:
you have to do it both before AND after, I think. I have to look at it/walk through it again, and I'll do so when I fix the reference to _templates so the values make sense, but the layered update to _mod (as a local variable before it is applied) needs care.

Reply:
sounds good! I think what I did was equivalent because of where the normalizeMods calls get invoked, but I definitely appreciate having another set of eyes on it